Telemetry/CorePing.swift

```diff
@@ -93,6 +93,10 @@ class CorePing: TelemetryPing {
             SearchTelemetry.resetCount(profile.prefs)
         }
 
+        if let chosenEmailClient = self.prefs.stringForKey(PrefsKeys.KeyMailToOption) {
+            out["mailClient"] = chosenEmailClient
```

Comment:
Can you change this to `defaultMailClient` ? This is what we already submitted to the Telemetry people to include in the dashboarding.